sklearn/decomposition/_lda.py

```diff
@@ -539,10 +544,12 @@ def partial_fit(self, X, y=None):
             Document word matrix.
 
         y : Ignored
+            Not used, present here for API consistency by convention.
 
         Returns
         -------
-        self
+        self:
```

Comment:
```suggestion
        self
```

Reply:
`self : object` for consistency with other places in the package

Reply:
yes, but I don't think it's correct. The base class for all estimators is BaseEstimator not object, so I think there is no point in indicating the object dtype.

Reply:
ok